drivers/spi/spi_ll_stm32.h

```diff
@@ -81,11 +81,19 @@ static inline uint32_t ll_func_dma_get_reg_addr(SPI_TypeDef *spi, uint32_t locat
 #endif /* st_stm32h7_spi */
 }
 
-/* checks that DMA Tx packet is fully transmitted over the SPI */
-static inline uint32_t ll_func_spi_dma_complete(SPI_TypeDef *spi)
+/* checks that DMA Tx packet is fully transmitted over the SPI.
+ *
+ * partial_transfer_size should be set if TSIZE is in use and will be reloaded from TSER at the end
```

Comment:
Please mention on which devices this behavior applies (or at least that it is device specific) and why not required on others.